vuepress/docs/v7.0/docs/curate/ecr-uninstall-flow.md

```diff
@@ -1,24 +1,33 @@
-# Bundle uninstall
+# Bundle Upgrade, Downgrade, and Uninstall
+When an application bundle is installed in the Entando App Builder, or from the Entando Hub, the advantage is that you can upgrade, downgrade or uninstall at any time. You can upgrade the bundle as a whole, or when changes are made to a single component, you can upgrade just that component. 
```

Comment:
the second sentence reads haltingly because you're expecting "or" to introduce an alternative "whole," not a different kind of upgrade